modules/ts/src/ts_gtest.cpp

```diff
@@ -4054,7 +4054,7 @@ enum GTestColor {
   COLOR_YELLOW
 };
 
-#if GTEST_OS_WINDOWS && !GTEST_OS_WINDOWS_MOBILE
+#if GTEST_OS_WINDOWS && !GTEST_OS_WINDOWS_MOBILE && !defined WINRT
```

Comment:
I believe you can add this code near GTEST_OS_WINDOWS_MOBILE detection:
```
#ifdef WINRT
// It is not the same things, but it makes GTest happy
#define GTEST_OS_WINDOWS_MOBILE 1
#endif
```

Is there exist fresh GTest package that supports WINRT?